lib/functions/date/date.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Formats a date string in ISO format (YYYY-MM-DD) to Norwegian format (DD.MM.YYYY)
+ * @param dateString - Date string in YYYY-MM-DD format
+ * @param locale - Locale to use for formatting (defaults to 'no-NO')
+ * @returns Formatted date string in Norwegian format
```

Comment:
Did you mean to export this for external use? If so, they are missing from `index.ts` in functions directory.